LoopBack/LBPersistedModel.h

```diff
@@ -7,12 +7,13 @@
 
 #import "LBModel.h"
 
+
+/**
+ * Blocks of this type are executed for a successful method invocation that returns
+ * an LBPersistedModel instance as a callback argument.
+ */
 @class LBPersistedModel;
 typedef void (^LBPersistedModelObjectSuccessBlock)(LBPersistedModel *model);
-typedef void (^LBPersistedModelArraySuccessBlock)(NSArray *array);
-typedef void (^LBPersistedModelVoidSuccessBlock)();
-typedef void (^LBPersistedModelBoolSuccessBlock)(BOOL boolean);
-typedef void (^LBPersistedModelNumberSuccessBlock)(NSInteger number);
```

Comment:
I didn't care about backward compatibility for those since those typedefs are added by me on Oct. 29 as a part of CodeGen support.  It seems highly unlikely that someone already depends on the typedefs.  How about make the above change and consider backward compatibility when we get any Issue on it?